src/core/server/saved_objects/migrationsv2/integration_tests/actions.test.ts

```diff
@@ -0,0 +1,449 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ElasticsearchClient } from '../../../';
+import { InternalCoreStart } from '../../../internal_types';
+import * as kbnTestServer from '../../../../test_helpers/kbn_server';
+import { Root } from '../../../root';
+import { SavedObjectsRawDoc } from '../../serialization';
+import {
+  bulkIndex,
+  cloneIndex,
+  createIndex,
+  fetchIndices,
+  reindex,
+  searchForOutdatedDocuments,
+  SearchResponse,
+  setWriteBlock,
+  updateAliases,
+  waitForReindexTask,
+  ReindexResponse,
+  waitForPickupUpdatedMappingsTask,
+  pickupUpdatedMappings,
+  UpdateByQueryResponse,
+} from '../actions';
```

Comment:
don't see tests for
- updateAndPickupMappings
- searchForOutdatedDocuments

Reply:
waiting for feedback from ES team on how to reproduce partial search results, but added the tests I could